### DISCUSSION TITLE
Github Activity Turned Red

### DISCUSSION THREAD

**Discussion by Arun-Kumar-Rathod** [community/community#37832] [Other Feature Feedback, Questions, & Ideas] [answered]
Today I was checking my GitHub profile, and I was surprised to see that my activity overview graph has turned red and, yes,
 I don't have any extensions that show this color. So if anyone knows something like it is a bad sign, please do let me know. Thank you :)

![g](https://user-images.githubusercontent.com/78579344/198907664-3aeb0ced-a12e-43e6-a4bc-4b8cc89e4814.png)

**Comment by radar-blip** [comment]
Don't know what would cause that to happen but it looks normal (green) from my side.

![graph](https://user-images.githubusercontent.com/39196767/198907961-4f5ed89e-04ae-4db9-aaef-b907654ea063.png)

**Comment by airtower-luna** [comment, marked as answer]
Look a little to the left for the explanation. @linuxlawson I assume the difference is that in the USA it's not the 31st *yet*. :grin: 

![grafik](https://user-images.githubusercontent.com/5814919/198907990-057e8cc6-7ab1-4f21-b51c-12d7587684f0.png)

**Reply by radar-blip** [reply]
Nope, not yet. Almost though (4 hours).

**Reply by Arun-Kumar-Rathod** [reply]
Thank you. my attention didn't go there ;)
